CMakeLists.txt

```diff
@@ -1,94 +1,48 @@
-cmake_minimum_required(VERSION 2.8.12)
+###########################################################################
+#                                                                         #
+# Note: The bulk of the build system is located in the cmake/ directory.  #
+#       This file only contains the specializations for this particular   #
+#       project. Most likely you are interested in editing one of these   #
+#       files instead:                                                    #
+#                                                                         #
+#       dune.module                              Name and version number  #
+#       CMakeLists_files.cmake                   Path of source files     #
+#       cmake/Modules/${project}-prereqs.cmake   Dependencies             #
+#                                                                         #
+###########################################################################
+
+# Mandatory call to project
 project(opm-parser CXX)
 
-file(STRINGS dune.module dm)
-if ("${dm}" MATCHES "Version: ([a-zA-Z0-9.-]*);")
-    string(REGEX MATCHALL "[a-z0-9]+" vers ${CMAKE_MATCH_1})
-    list(GET vers 0 opm-parser_VERSION_MAJOR)
-    list(GET vers 1 opm-parser_VERSION_MINOR)
+cmake_minimum_required(VERSION 2.8)
 
-    message(STATUS "opm-parser version set to "
-                   ${opm-parser_VERSION_MAJOR} "."
-                   ${opm-parser_VERSION_MINOR} " in dune.module")
-else ()
-    message(SEND_ERROR "Could not read version from dune.module")
-endif ()
-
-include(CTest)
-include(GNUInstallDirs)
-include(TestCXXAcceptsFlag)
-
-if(NOT CMAKE_BUILD_TYPE AND NOT CMAKE_CONFIGURATION_TYPES)
-    message(STATUS "No build type specified - defaulting to 'Release'.")
-    set(CMAKE_BUILD_TYPE "Release" CACHE STRING "Choose build type." FORCE)
-    set_property(CACHE CMAKE_BUILD_TYPE PROPERTY
-                 STRINGS "Release" "Debug" "RelWithDebInfo" "MinSizeRel")
-endif()
-
-option(BUILD_TESTING "Build test applications by default?"          ON)
-option(USE_RUNPATH   "Embed dependency paths in installed library"  ON)
 option(SIBLING_SEARCH "Search for other modules in sibling directories?" ON)
+option(BUILD_TESTING "Build test applications by default?" ON)
```

Comment:
Seems like ewoms, opm-parser, and opm-output are the only modules making `BUILD_TESTING` visible in CMakeCache.txt. In the other modules this option is not exposed.

Reply:
copy-paste issue from opm-output. indeed, this is in generic part of build system where it is not exposed as an option (i think it should though but generic not in this module). removed.